### ISSUE TITLE
Copyright date out of date on --version

### ISSUE THREAD

**Issue by jidanni** [bbidulock/icewm#453]
```
$ icewm --version
IceWM 1.6.6, Copyright 1997-2003 Marko Macek, 2001 Mathias Hasselmann.
```
Shouldn't this be updated?

**Comment by gijsbers** [comment]
#285

**Comment by rubyFeedback** [comment]
This issue here should probably be closed, for #285 to discuss and continue (if necessary), since here it is essentially a duplicate.
